news/2018-09-18-the-qat-gazette-august-2018.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "The QAT Gazette: August 2018"
+date: 2018-09-18 10:00:00 +0000
+---
+
+Modding discussion feature requests, potential changes to hybrid mode mapsets, and an update on awaited Beatmap Nominator applications — all your mapping and modding news from the past month is inside!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+## Opening
+by [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+Hello again!
+
+The summer vacation is coming to an end, for most of us heralding the return to school, university or work. Whether this inspires excitement, dread, or any emotion in-between, a healthy dose of mapping and modding news updates is surely just what you need! If you're here to keep up to date with Beatmap Nominator rule changes, new and upcoming Ranking Criteria updates, or recent meetings of the Quality Assurance Team, we've got you covered. Scroll down to find your own little slice of information heaven!
+
+Before we begin though, please join me in showing the utmost appreciation for some of our longest serving contributors in the Beatmap Nominators — [alienflybot](https://osu.ppy.sh/users/636114), [Kagetsu](https://osu.ppy.sh/users/6203841), [sahuang](https://osu.ppy.sh/users/5318910) and [Strategas](https://osu.ppy.sh/users/2971837) (two years of service each), [Kibbleru](https://osu.ppy.sh/users/3193504) (three years) and [Gero](https://osu.ppy.sh/users/1467715), [jonathanlfj](https://osu.ppy.sh/users/270377), [Natsu](https://osu.ppy.sh/users/1953876) and [smallboat](https://osu.ppy.sh/users/243049), each with a staggering **four years** of cumulative service.
+
+From the Quality Assurance Team, we would also like to recognize the incomparable efforts of [Deif](https://osu.ppy.sh/users/318565) and [p3n](https://osu.ppy.sh/users/123703), who have served admirably in their respective fields for an incredible **four years**.
+
+It is no exaggeration to state that their tireless efforts have helped shape osu!'s mapping and modding communities into the successful force they are today, having assisted and mentored a countless number of mappers along the way. Each will receive a profile badge advertising their length of service which they are encouraged to display with pride!
```

Comment:
I'm not sure "advertising" is the right word here. Think more "recognising" or "acknowledging"

Reply:
Acknowledging is good, changed